functions/venue.js

```diff
@@ -406,20 +407,6 @@ const dataOrUpdateKey = (data, updated, key) =>
     typeof updated[key] !== "undefined" &&
     updated[key]);
 
-/** Add a user to the list of admins
- *
- * @param {string} newAdminId
- */
-const addAdmin = async (newAdminId) => {
```

Comment:
[NB] moved addAdmin to separate utility file as to not create direct dependency between auth.js and venue.js files that both require() it